<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFinder</class>
 <widget class="QDockWidget" name="TextFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>103</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QToolButton" name="prev">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="next">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>10</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="ReplaceAll">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>65</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string>Replace All</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Replace">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="AdvancedSearch">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Lunch Advanced Search</string>
    </property>
    <property name="text">
     <string>Advanced...</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="findEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="replaceEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
